### PR TITLE
Feature/19/countries geojson

### PR DESCRIPTION
**Pull request by cpapalaz**
# Description
code to create json from CSV which filters out countries without mangroves 

Fixes # 19

## Type of change

- [ ] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

### CODE REVIEW

File: countries-geojson/countries_geojson.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+from re import M
+from geojson import Feature, FeatureCollection, Point
+
+csvFilePath = "/Users/cpapalaz/Desktop/Mangrove_Countries.csv"
+jsonFilePath = "/Users/cpapalaz/Desktop/Mangrove_Countries.json"
+
+#read csv file and add to data
+features = []
+with open(csvFilePath, newline='', encoding='latin1') as csvfile:
+    print(csvfile)
```
Comment (JBurkinshaw) [inline countries-geojson/countries_geojson.py:11]: The `print` statement isn't required. Can you remove it? (I know I left it in my example 😅)

File: countries-geojson/countries_geojson.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import csv, json
+from re import M
```
Comment (JBurkinshaw) [inline countries-geojson/countries_geojson.py:2]: Is this import being used anywhere? If not, can you remove it?

File: countries-geojson/countries_geojson.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+                properties = {
+                    'country': Country,
+                    'mangroves': Mangroves
+                }
+            )
+        )
+        else:
+            pass
+
+
```
Comment (JBurkinshaw) [inline countries-geojson/countries_geojson.py:31]: New lines not required here.

**Review by JBurkinshaw — CHANGES_REQUESTED** [review]
Nice work. The output looks good - I switched to using it with the map I have seamlessly.

There are a few comments in the code and can you rename it to `mangrove_countries.json` and move it to `gmw-user/mrtt-ui/src/data` to keep it in line with the work I'm doing please?

I'd also suggest formatting the code with [black](https://github.com/psf/black) to make sure everything is following formatting best practices.

File: countries-geojson/countries_geojson.py [hunk 4]
```diff
@@ -0,0 +1,35 @@
+import csv, json
+from re import M
+from geojson import Feature, FeatureCollection, Point
+
+csvFilePath = "/Users/cpapalaz/Desktop/Mangrove_Countries.csv"
+jsonFilePath = "/Users/cpapalaz/Desktop/Mangrove_Countries.json"
```
Comment (JBurkinshaw) [inline countries-geojson/countries_geojson.py:6]: [Best practice](https://peps.python.org/pep-0008/#function-and-variable-names) in python would be to name variables with snake case (words lowercase and separated by underscores).

Hard coding paths which are specific to your environment isn't ideal. Can you make these relative paths? I know relative paths worked in my example which I wrote in a Jupyter notebook but it's not always straightforward in python. If the suggestion below doesn't work, maybe try something like [this](https://stackoverflow.com/a/918178/3704248).

```suggestion
csv_file_path = "Mangrove_Countries.csv"
json_file_path = "../mrtt-ui/src/data/mangrove_countries.json"
```

File: countries-geojson/countries_geojson.py [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+        (
+            Country,
+            Mangroves,
+            Latitude,
+            Longitude,
+            MIN_POINT_X,
+            MAX_POINT_X,
+            MIN_POINT_Y,
+            MAX_POINT_Y,
+        ) = row
```
Comment (JBurkinshaw) [inline countries-geojson/countries_geojson.py:22]: Nice

**Review by JBurkinshaw — APPROVED** [review]

**Comment by ghelobytes** [comment]
@JBurkinshaw  @cpapalaz 

Do we want to create a parent folder for these scripts?
I have written some python script to convert the other `csv` to `json` and I can add those here.

Maybe something like this:

```
root/
├── mrtt-ui/
├── scripts/
│   ├── countries-geojson/
│       ├── README.md
│       ├── countries_geojson.py
│   ├── causes-tree/
│       ├── README.md
│       ├── causes_tree.py
│   ├── species-binary/
│       ├── README.md
│       ├── species_binary.py
```

**Comment by JBurkinshaw** [comment]
> @JBurkinshaw @cpapalaz
> 
> Do we want to create a parent folder for these scripts? I have written some python script to convert the other `csv` to `json` and I can add those here.
> 
> Maybe something like this:
> 
> ```
> root/
> ├── mrtt-ui/
> ├── scripts/
> │   ├── countries-geojson/
> │       ├── README.md
> │       ├── countries_geojson.py
> │   ├── causes-tree/
> │       ├── README.md
> │       ├── causes_tree.py
> │   ├── species-binary/
> │       ├── README.md
> │       ├── species_binary.py
> ```

Sounds good to me. Can you go ahead with the restructure and open a PR?

**Comment by JBurkinshaw** [comment]
Closes #19

**Pull request merged**